packages/svelteui-core/src/components/Table/Table.d.ts

```diff
@@ -0,0 +1,10 @@
+import { HTMLAttributes } from 'svelte/elements';
+import { DefaultProps, SvelteUINumberSize } from '$lib/styles';
+
+export interface TableProps extends DefaultProps, HTMLAttributes<HTMLElement> {
```

Comment:
So that it applies the correct type to the element prop and also allows to pass table attributes as props:

```suggestion
export interface TableProps<T = HTMLTableElement> extends DefaultProps<T>, HTMLTableAttributes {
```